Plan/nukkit/src/main/java/com/djrapitops/plan/addons/placeholderapi/NukkitPlaceholderRegistrar.java

```diff
@@ -56,27 +60,36 @@ public NukkitPlaceholderRegistrar(
 
     public void register() {
         PlaceholderAPI api = PlaceholderAPI.getInstance();
-        placeholders.getPlaceholders().forEach((name, loader) ->
-                api.visitorSensitivePlaceholder(name, (player, params) -> {
-                            try {
-                                return loader.apply(getPlayer(player), params.get());
-                            } catch (Exception e) {
-                                errorLogger.warn(e, ErrorContext.builder().related("Registering PlaceholderAPI").build());
-                                return null;
-                            }
-                        }
-                ));
+        placeholders.getPlaceholders().forEach((name, loader) -> api.builder(name, Serializable.class)
+                .visitorLoader(entry -> {
+                    try {
+                        return loader.apply(
+                                getPlayer(entry.getPlayer()),
+                                entry.getParameters().getAll().stream()
```

Comment:
This parameter stream mapping can be extracted to its own method to avoid duplicate code. Naming the function might make the code more readable.